### PR TITLE
python3.pkgs.hbmqtt: needs setuptools at runtime

### PR DESCRIPTION
**Pull request by Mic92**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/hbmqtt/default.nix [hunk 1]
```diff
@@ -12,7 +12,9 @@ buildPythonPackage rec {
     sha256 = "1n9c8yj11npiq9qxivwmfhib1qkjpcyw42a7q0w641bdrz3x6r37";
   };
 
-  propagatedBuildInputs = [ transitions websockets passlib docopt pyyaml ];
+  propagatedBuildInputs = [
+    transitions websockets passlib docopt pyyaml setuptools
```
Comment (Mic92) [inline pkgs/development/python-modules/hbmqtt/default.nix:16]: check with `hbmqtt_pub`.

Comment (SuperSandro2000) [reply pkgs/development/python-modules/hbmqtt/default.nix:16]:
```
pythonImportsCheck = [ "hbmqtt_pub" ];
```

Comment (Mic92) [reply pkgs/development/python-modules/hbmqtt/default.nix:16]: `hbmqtt_pub` is an executable.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 103058 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python37Packages.hbmqtt</li>    <li>python38Packages.hbmqtt</li>  </ul></details>

**Pull request merged**